src/nuklear_popup.c

```diff
@@ -134,7 +134,11 @@ nk_nonblock_begin(struct nk_context *ctx,
     } else {
         /* close the popup if user pressed outside or in the header */
         int pressed, in_body, in_header;
+#ifdef NK_BUTTON_TRIGGER_ON_RELEASE
+		pressed = nk_input_is_mouse_released(&ctx->input, NK_BUTTON_LEFT);
```

Comment:
Feel free to correct the indentation (we're using spaces) :wink:.

Reply:
Done